plugins/modules/sonic_mclag.py

```diff
@@ -752,6 +860,14 @@
 # Mclag System Mac     : 00:00:00:11:11:11
 # Gateway Mac          : 00:00:00:12:12:12
 #
+# Backup Keepalive Session Information:
+# -----------------------------------
+# Session Vrf          : Default
+# Session Status       : down
+# Source Address       : 131.131.131.131
+# Peer Address         : 
```

Comment:
Please delete the trailing space at the end of this line (868). It is causing the sanity tests to fail for this PR.

Reply:
Removed